cmd/build.go

```diff
@@ -0,0 +1,80 @@
+package cmd
+
+import (
+	"context"
+	"log"
+	"os"
+	"path"
+	"time"
+
+	"github.com/mitchellh/go-homedir"
+	"github.com/screwdriver-cd/sd-local/buildlog"
+	"github.com/screwdriver-cd/sd-local/config"
+	"github.com/screwdriver-cd/sd-local/launch"
+	"github.com/screwdriver-cd/sd-local/screwdriver"
+	"github.com/spf13/cobra"
+)
+
+const (
+	waitIO = 1
+)
+
+func newBuildCmd() *cobra.Command {
+	buildCmd := &cobra.Command{
+		Use:   "build [job name]",
+		Short: "Run screwdriver build of the specified job name.",
```

Comment:
How about "Run screwdriver build" as `Short` usage?

Reply:
I see.
I fixed it